doc/source/multi_run_plans.rst

```diff
@@ -0,0 +1,511 @@
+Multi-Run Plans
+===============
+
+This section is a brief tutorial on multi-run plans (introduced in Bluesky v1.6.0).
+A traditional single-run plan contains a set of instructions for running only one scan,
```

Comment:
We try to avoid using the word "scan" to mean "run" because:

* Some experiments aren't really scanning anything, and people in the past have been confused on that point.
* Sometimes a run is scanning multiple axis, such as `grid_scan`, so wording like this can be confusing.

"Run" is a pretty vague term, but we have never come up with a better one that sufficiently conveys how general it is. This language the [the event-model documentation on our Data Model](https://blueskyproject.io/event-model/data-model.html) may be useful:

> The scope of a “run” is left to the instrument team or individual scientist. It is quite generic: a set of Documents generated by following a given sequence of instructions. This might encompass a single short measurement (say, “a count”) or a multi-step procedure such as a raster scan. A run should represent a set of individual measurements that will be collected or processed together.

I think every instance of the word "scan" here and below should be changed to "run" or "plan", depending on the situation.

Reply:
It seems like 'run' is a vague term when speaking about the structure of data acquisition system. But technically from the point of view of Bluesky it is a sequence of messages that starts with `open_run` and ends with `close_run`. The data acquired during run execution will be kept together and could be identified by scan ID and UID. A plan may contain no defined run (for example just move motors). Would that be correct?

I was thinking if I should use 'scan' or 'run' and decided to use 'scan', but I guess 'run' is more accurate word.

Reply:
Yeah, I agree that "run" is a vague term. I think it's the right choice for internal consistency with the object names in our libraries (`open_run`) and the rest of the documentation, but I agree it's not ideal. I sometimes use the word "scan" when speaking to certain audiences, but for a general audience, "run" is more precise.